app/views/articles/_actions.html.erb

```diff
@@ -32,10 +32,11 @@
       <button id="article-show-more-button" aria-controls="article-show-more-dropdown" aria-expanded="false" aria-haspopup="true" class="dropbtn crayons-btn crayons-btn--ghost-dimmed crayons-btn--icon-rounded" aria-label="Share post options">
         <%= inline_svg_tag("overflow-horizontal.svg", aria_hidden: true, class: "dropdown-icon crayons-icon", title: "More...") %>
       </button>
-      <div id="article-show-more-dropdown" class="crayons-dropdown p-1 right-1 left-1 s:left-auto bottom-100 m:bottom-auto m:top-0 m:right-auto m:left-100 fs-base">
-        <clipboard-copy for="article-copy-link-input" aria-live="polite" aria-controls="article-copy-link-announcer" class="dropdown-link-row">
-          <div class="flex items-center">
-            <input type="text" id="article-copy-link-input" value="<%= article_url(@article) %>" aria-label="Post URL" class="crayons-textfield" readonly />
+
+      <div id="article-show-more-dropdown" class="crayons-dropdown side-bar left-1 s:left-auto m:left-100">
+        <clipboard-copy for="article-copy-link-p" aria-live="polite" aria-controls="article-copy-link-announcer" class="dropdown-link-row">
+          <div class="flex justify-between cursor-pointer dropdown-link-row crayons-link crayons-link--block" tabindex="0">
+            <p class="fw-bold" id="article-copy-link-p" aria-label="Post URL" data-postUrl="<%= article_url(@article) %>">Copy Post URL</p>
             <%= inline_svg_tag("copy.svg", aria: true, id: "article-copy-icon", class: "crayons-icon mx-2 shrink-0", title: "Copy article link to the clipboard") %>
           </div>
           <div id="article-copy-link-announcer" class="crayons-notice crayons-notice--success my-2 p-1" aria-live="polite" hidden>Copied to Clipboard</div>
```

Comment:
This markup needs a bit of rework so that we rely on semantic HTML elements.

- I think we can change `clipboard-copy` to a `div`. As far as I can tell we are not using the features of the `clipboard-copy-element` in this area (instead using our own custom code). We also don't want `aria-live` or `aria-controls` on this element. We have `aria-live` on the `article-copy-link-announcer` which is sufficient to announce the successful copy event to screen reader users. 
- The 'clickable' element should be a `button`, rather than a `p` and an `svg`. The button doesn't need an `aria-label` because the button text conveys the purpose already. The SVG is decorative and should be hidden with `aria_hidden: true`. We can then remove the `tabindex="0"` and additional keypress listener since buttons have all this behaviour reliably baked-in 🙂